src/view/treeNodes/pullRequestNode.ts

```diff
@@ -504,265 +571,206 @@ export class PRNode extends TreeNode {
 		return '';
 	}
 
-	private findMatchingFileNode(uri: vscode.Uri): InMemFileChangeNode {
-		const params = fromPRUri(uri);
-
-		if (!params) {
-			throw new Error(`${uri.toString()} is not valid PR document`);
-		}
-
-		const fileChange = this._fileChanges.find(change => change.fileName === params.fileName);
+	// #endregion
 
-		if (!fileChange) {
-			throw new Error('No matching file found');
-		}
+	// #region comment
+	public async createOrReplyComment(thread: vscode.CommentThread) {
+		if (await this._prManager.authenticate() && this.commentController!.inputBox !== undefined) {
+			if (thread.comments.length) {
+				let comment = thread.comments[0] as (vscode.Comment & { _rawComment: Comment });
+				const rawComment = await this._prManager.createCommentReply(this.pullRequestModel, this.commentController!.inputBox!.value, comment._rawComment);
 
-		if (fileChange instanceof RemoteFileChangeNode) {
-			throw new Error('Comments not supported on remote file changes');
+				thread.comments = [...thread.comments, convertToVSCodeComment(rawComment!, undefined)];
```

Comment:
The edit and delete commands are not being added here, we should have another method that will create comments when the thread already exists